src/ms-graph/__tests__/__recordings__/fetchOrganization_3927641460/recording.har

```diff
@@ -0,0 +1,305 @@
+            },
+            {
+              "name": "x-ms-client-request-id",
+              "value": "aa96fed3-39d4-4da4-8e7a-07ddd25c649f"
+            },
+            {
+              "name": "content-length",
+              "value": "202"
+            },
+            {
+              "name": "host",
+              "value": "login.microsoftonline.com"
+            }
+          ],
+          "headersSize": 435,
+          "httpVersion": "HTTP/1.1",
+          "method": "POST",
+          "postData": {
+            "mimeType": "application/x-www-form-urlencoded",
+            "params": [],
+            "text": "response_type=token&grant_type=client_credentials&client_id=a8626c1e-191d-4e8f-9cbc-a4a6e85104ac&client_secret=L3X8Q~BO57QkoAsXMSkevrmVR2qiNh.qKEuzucAt&scope=https%3A%2F%2Fgraph.microsoft.com%2F.default"
+          },
+          "queryString": [],
+          "url": "https://login.microsoftonline.com/5a721b05-53ed-4ed9-be02-aed28f11edbd/oauth2/v2.0/token"
+        },
+        "response": {
+          "bodySize": 1789,
+          "content": {
+            "mimeType": "application/json; charset=utf-8",
+            "size": 1789,
+            "text": "{\"token_type\":\"Bearer\",\"expires_in\":3599,\"ext_expires_in\":3599,\"access_token\":\"eyJ0eXAiOiJKV1QiLCJub25jZSI6IlQ2TUpMYy1CRHMtNjdVbU5rcy1KZmJuUzJ5U3oxa0xpZzd0c3hfTGUxV3MiLCJhbGciOiJSUzI1NiIsIng1dCI6ImpTMVhvMU9XRGpfNTJ2YndHTmd2UU8yVnpNYyIsImtpZCI6ImpTMVhvMU9XRGpfNTJ2YndHTmd2UU8yVnpNYyJ9.eyJhdWQiOiJodHRwczovL2dyYXBoLm1pY3Jvc29mdC5jb20iLCJpc3MiOiJodHRwczovL3N0cy53aW5kb3dzLm5ldC81YTcyMWIwNS01M2VkLTRlZDktYmUwMi1hZWQyOGYxMWVkYmQvIiwiaWF0IjoxNjU1MjEyMzE3LCJuYmYiOjE2NTUyMTIzMTcsImV4cCI6MTY1NTIxNjIxNywiYWlvIjoiRTJaZ1lIaThWMWhmdm5tTGRtT3JiTGJqOGxZVEFBPT0iLCJhcHBfZGlzcGxheW5hbWUiOiJkZWZlbmRlckFwcDEiLCJhcHBpZCI6ImE4NjI2YzFlLTE5MWQtNGU4Zi05Y2JjLWE0YTZlODUxMDRhYyIsImFwcGlkYWNyIjoiMSIsImlkcCI6Imh0dHBzOi8vc3RzLndpbmRvd3MubmV0LzVhNzIxYjA1LTUzZWQtNGVkOS1iZTAyLWFlZDI4ZjExZWRiZC8iLCJpZHR5cCI6ImFwcCIsIm9pZCI6ImU5NzNhNzA4LTVjODQtNDU3MS1hNmQyLTM2OGJiYjE2ZTJlZSIsInJoIjoiMC5BVlVBQlJ0eVd1MVQyVTYtQXE3U2p4SHR2UU1BQUFBQUFBQUF3QUFBQUFBQUFBQ0lBQUEuIiwicm9sZXMiOlsiVXNlci5SZWFkV3JpdGUuQWxsIiwiRGlyZWN0b3J5LlJlYWRXcml0ZS5BbGwiLCJEaXJlY3RvcnkuUmVhZC5BbGwiLCJVc2VyLlJlYWQuQWxsIiwiT3JnYW5pemF0aW9uLlJlYWQuQWxsIiwiUG9saWN5LlJlYWQuQWxsIiwiUmVwb3J0cy5SZWFkLkFsbCJdLCJzdWIiOiJlOTczYTcwOC01Yzg0LTQ1NzEtYTZkMi0zNjhiYmIxNmUyZWUiLCJ0ZW5hbnRfcmVnaW9uX3Njb3BlIjoiQVMiLCJ0aWQiOiI1YTcyMWIwNS01M2VkLTRlZDktYmUwMi1hZWQyOGYxMWVkYmQiLCJ1dGkiOiJ2OHdoRFY5QVFVV0JfcUNXWmt3NEFBIiwidmVyIjoiMS4wIiwid2lkcyI6WyIwOTk3YTFkMC0wZDFkLTRhY2ItYjQwOC1kNWNhNzMxMjFlOTAiXSwieG1zX3RjZHQiOjE2NTI3NzM2ODF9.J3_UaOTMoxYAWw6YfAM3GNpZv7ZgZEbQkAlZ_YUjUCWsTihWgXyHNs-j5VArtRzpf0HLJRfQw7b7UrDmJPbZJfMsMRzUPwCz5AMz7M6pUnReNRRP2qFiLXkm2NBFWjbunYSSLO29mDCWvTI2mJrByvdqb5Ksv7g-UWur5E1IzqtRKgD1D03orCNLRHP7xYBV38mmuS2TaIF8zsnI5BHG_e8sM__oVWXlQy25IcaEA0eJQNRQOBFqw0mYy60oKlyjZD7pFjI0yB3DDV5K68k5H4ppjlgSdrxXrkvoa61epzyZoYa5QJm01hIWuGiYyO7cr6eA6EVZVEACVFanSsVEig\"}"
```

Comment:
It is important to ensure that no credentials are recorded in any commit to the repo. See https://github.com/JupiterOne/graph-microsoft-endpoint-defender/blob/8b834b13321bfbaaed184d1f506cf726d131c5da/test/recording.ts for the place to manage this, with https://github.com/JupiterOne/graph-microsoft-365/blob/a2cceb9734bf9715ccbc87a53c0a5e1fbc8dc96f/test/recording.ts providing example code.